ctdcal/oxy_fitting.py

```diff
@@ -7,8 +7,8 @@
 """
 
 import sys
-sys.path.append('/Users/k3jackson/p06e/ctd_proc')
-sys.path.append('/Users/k3jackson/odf-ctd-proc/ctdcal/')
+sys.path.append('/ctd_proc')
```

Comment:
Need to fix absolute links - they will look at the root of the harddrive, instead of looking in the directory the software is installed.